Source/UI/NewUI/CommonUI/View/TextFieldView.swift

```diff
@@ -27,6 +27,14 @@ class TextFieldView: UIView {
         return view
     }()
 
+    private(set) lazy var mandatoryLabel: UILabel? = {
+        let view = UILabel()
+        view.numberOfLines = 0
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
```

Comment:
I would avoid hardcoding these colours, especially as you will need them customised and presumably the label doesn't require a separate background colour for itself only (haven't seen a design so may as well be).

Reply:
Fixed in this [PR](https://github.com/checkout/frames-ios/pull/168)